### PR TITLE
Add info for 3rd party repos

### PR DESCRIPTION
**Pull request by chuckersjp**
[OCPBUGS-26552]

Version(s):

Docs version from OCP 4.9 onwards
Issue:

https://github.com/openshift/openshift-docs/issues/50471

Link to docs preview:

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by michaelryanmcneill** [comment]
@chuckersjp I'd like to run this by a few others before approving it. Will circle back tomorrow.

**Comment by chuckersjp** [comment]
> @chuckersjp I'd like to run this by a few others before approving it. Will circle back tomorrow.

Sure thing.  This was actually something some of our friends up North pointed out and asked for.  :grimacing:

**Comment by michaelryanmcneill** [comment]
@chuckersjp I figured it was 😂

**Review by michaelryanmcneill — APPROVED** [review]
LGTM

**Comment by michaelryanmcneill** [comment]
/lgtm

Not sure if with Prow I need to also run this command @kalexand-rh. But either way this is safe to get merged and cherrypicked.

File: modules/osd-aws-privatelink-firewall-prerequisites.adoc [hunk 1]
```diff
@@ -309,6 +309,24 @@ OR
...
+| 443
+| Required for all third-party-images and certified operators.
+
+|`rhc4tp-prod-z8cxf-image-registry-us-east-1-evenkyleffocxqvofrk.s3.dualstack.us-east-1.amazonaws.com`
+| 443
+| Provides access to container images hosted on `registry.connect.redhat.com`
+
+|`oso-rhc4tp-docker-registry.s3-us-west-2.amazonaws.com`
+| 443
+| Required for Sonatype Nexus, F5 Big IP operators.
```
Comment (jeana-redhat) [inline modules/osd-aws-privatelink-firewall-prerequisites.adoc:327]: Not 100% sure on the naming of either Operator, suggestions based on [this page for Nexus](https://support.sonatype.com/hc/en-us/articles/4757606085523-How-to-install-Nexus-Pro-using-Nexus-Operator-in-OpenShift) and [this page for F5](https://community.f5.com/t5/technical-articles/using-f5-big-ip-controller-operator-for-openshift/ta-p/288220).

```suggestion
| Required for the Nexus Repository Operator and F5 BIG-IP Operator.
```

File: modules/osd-aws-privatelink-firewall-prerequisites.adoc [hunk 2]
```diff
@@ -309,6 +309,24 @@ OR
...
 |===
 
+. Allowlist the following URLs for optional third-party content:
++
+[cols="6,1,6",options="header"]
+|===
+|Domain | Port | Function
+|`registry.connect.redhat.com`
+| 443
+| Required for all third-party-images and certified operators.
```
Comment (jeana-redhat) [inline modules/osd-aws-privatelink-firewall-prerequisites.adoc:319]:
```suggestion
| Required for all third-party-images and certified Operators.
```

**Review by jeana-redhat — COMMENTED** [review]
Thanks for the contribution! I have left a few comments around terminology. The third-party Operators listed at the end I'm not very confident about, but as long as we can have something authoritative (maybe how it appears written wherever the customer would find it?) we should be good.

/remove-label peer-review-in-progress
/remove-label peer-review-needed
/label peer-review-done

**Comment by chuckersjp** [comment]
@jeana-redhat 

Thank you for the review.

I pretty much copied what is already in our docs.  See here: https://docs.openshift.com/container-platform/4.9/installing/install_config/configuring-firewall.html

I don't think I am particularly committed to either wording and I can see arguments for both sides.  A small `o` to me would make me think more of a generic term referring to the object itself rather than as a proper name which would require a capital `O` as you point out.  That appears to be used for the `Nexus Repository Operator` but it is is referred to as `F5 BIG-IP Controller` in your links.

**Comment by jeana-redhat** [comment]
> I pretty much copied what is already in our docs. See here: https://docs.openshift.com/container-platform/4.9/installing/install_config/configuring-firewall.html

Thanks, that's helpful context. For the sake of consistency, I think this is fine as-is, and folks can take a look at the terminology across the board at another time if desired. Updating the existing content would definitely be out of scope of this PR.

LGTM for peer review and merge review. Are you happy with it/ready for merge?

**Comment by chuckersjp** [comment]
I am good with it but I am not certain you are asking me or if the decision is even up to me.

**Comment by jeana-redhat** [comment]
> I am good with it but I am not certain you are asking me or if the decision is even up to me.

Sorry for not being clear! It LGTM from the perspective of a merge reviewer, but I didn't want to merge without confirming that you had no further changes you wanted to make. Sounds like you are happy with it from your side :+1:

**Pull request merged**